src/internet_identity/tests/integration/openid.rs

```diff
@@ -224,12 +410,12 @@ fn cannot_register_with_faulty_jwt() {
 
     let canister_id = setup_canister(&env);
 
-    let (_jwt, salt, _claims, test_time, test_principal, _test_authn_method) = openid_test_data();
+    let (_jwt, salt, _claims, test_time, test_principal, _test_authn_method) =
```

Comment:
`#[should_panic]` should be used together with `expected`, e.g.

```
#[should_panic(expected = "missing field `signals_end`")]
``` 

Otherwise, the test isn't necessarily testing what it claims, as any panic in this code (even an undesirable one) would lead to the test succeeding.

No change required since it wasn't added in this PR, just wanted to mention.

Reply:
It's a substring of the panic message